client/doc/onboarding.md

```diff
@@ -61,6 +61,30 @@ Go directly to the given step and trigger required events. Useful for go back in
 Returns a step by its name.
 
 ### Step
+
+#### isActive(user)
+* `user`: JS Object
+
+Returns true if the step has to be active for the given `user`. Returns `true` by default.
```

Comment:
I'm not sure that your first sentence his usefull. It seems obvious according to the method name. Furthermore your exemple below speaks for itself.
Precise default value by the way is great 👍